releasetasks/templates/mobile/common_extras.yml.tmpl

```diff
@@ -14,6 +14,6 @@ build_props:
     revision: {{ revision }}
```

Comment:
I'm pretty sure we don't actually ship any mobile stuff through Balrog at this point (except Nightly, which won't be covered by multiple signoffs). I guess it's better to update this than not...but maybe code around this should be removed at some point?